src/xdocs/config_javadoc.xml

```diff
@@ -176,7 +176,8 @@
           Note that only inheritable items will allow the
           <code>{@inheritDoc}</code> tag to be used in place
           of comments.  Static methods at all visibilities, private non-static
-          methods and constructors are not inheritable.
+          methods and constructors are not inheritable. we don't require a
+          full summary for {@inheritDoc}.
```

Comment:
Why you updated JavadocMethod documentation ?

your check is SummaryJavadoc -  http://checkstyle.sourceforge.net/config_javadoc.html#SummaryJavadoc

update should be ".... ontain phrases that are not recommended to use. Comments that contains only InheritDoc tag are skipped."